src/ostorlab/cli/dumpers.py

```diff
@@ -1,18 +1,17 @@
 """Module responsible for dumping data to different formats."""
 
 import abc
-import json
 import csv
-from typing import Dict
-
+import json
+import typing
```

Comment:
typing is an exception to class import per style guide. You can use from typing import List